### PR TITLE
Update README and add Heroku button

### PR DESCRIPTION
**Pull request by jwilander**
#### Summary
Update README

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,37 +1,48 @@
 # Mattermost
 
-Mattermost Team Edition is an open source, self-hosted Slack-alternative. 
+Mattermost is an open source, self-hosted Slack-alternative from [https://mattermost.org](https://mattermost.org).
 
-It's written in Golang and React and runs as a single Linux binary with either MySQL or Postgres. Every month on the 16th a new compiled version is released under an MIT license. 
+It's written in Golang and React and runs as a single Linux binary with MySQL or Postgres. Every month on the 16th [a new compiled version is released under an MIT license](https://www.mattermost.org/download/).
```
Comment (enahum) [inline README.md]: We have binaries for linux, osx and windows right?

**Comment by enahum** [comment]
+1 other than my comment on the binaries

File: README.md (hunk 1 above)
Comment (jwilander) [inline README.md]: Yes, but I think we only 'officially' support linux right now (hence all our install guides being for linux distros)

**Comment by hmhealey** [comment]
+1

**Pull request merged**